src/services/event.js

```diff
@@ -273,11 +277,83 @@ const makeMatchesFilters = (filters) => (event) => {
   return false;
 };
 
+/**
+ * Splits up multiple occurrences of events into individual events
+ * and re-sorts the array
+ *
+ * @param {Object[]} events the array of events to process
+ * @returns {Object[]} the flattened array of events after splitting multiple occurrences
```

Comment:
Giving the param and return value type `Object[]` is not very helpful. We have type definitions for events and event occurrences in this file. Maybe you should use/update those? Having accurate type definitions is especially helpful because it informs VSCode's intellisense and code completion.